auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -613,6 +685,9 @@ public boolean verifyEvidence(final Evidence evidence) throws RealmUnavailableEx
         }
 
         List<Credential> loadCredentials() throws RealmUnavailableException {
+            if (!verifyIntegrity()) {
```

Comment:
This should be called from `loadIdentityPrivileged`.

Reply:
Actually, looks like you've got a call to `verifyIntegrity` there already. This extra call here should be removed.